modules/cluster-logging-elasticsearch-tolerations.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-elasticsearch.adoc
+
+[id="cluster-logging-elasticsearch-tolerations_{context}"]
+= Using tolerations to control the Elasticsearch pod placement
+
+You can use taints and tolerations to ensure that cluster logging pods 
+run on specific nodes and that no other workload can run on those nodes.
+
+You apply tolerations to Elasticsearch pods through the Cluster Logging Custom Resource (CR)
+and apply taints to a node through the node specification. A taint on a node 
+instructs the node to repel all pods that do not tolerate the taint.
+
+By default, the Elasticsearch pods have the following toleration:
+
+[source, yaml]
+----
+tolerations:
+- effect: "NoExecute"
+  key: "node.kubernetes.io/disk-pressure"
+  operator: "Exists"
+----
+
+.Prerequisites
+
+* Cluster logging and Elasticsearch must be installed.
+
+.Procedure
+
+. Use the following command to add taints and tolerations for cluster logging:
++
+----
+$ oc adm taint nodes <node-name> <key>=<value>:<effect>
+----
++
+For example:
++
+----
+$ oc adm taint nodes node1 key1=logging:NoExecute
```

Comment:
Didn't we just do this? Do you put the pods on different nodes? Is there more guidance that we can share about how to direct them?

Reply:
The RFE addresses a request to include the cluster logging components in the customer's broader toleration policies. The dev user story seems to emphasize preventing other workloads from running on the cluster logging nodes. 
I added wording to this effect.